documentation/master.adoc

```diff
@@ -87,6 +87,12 @@ include::assemblies/assembly-monitoring-oc.adoc[leveloffset=+3]
 include::assemblies/assembly-monitoring-kube.adoc[leveloffset=+3]
 include::assemblies/assembly-using-qdstat.adoc[leveloffset=+3]
 
+=== Uninstalling {ProductName}
```

Comment:
@jenmalloy we're going to want to revisit master.adoc after next week because headings are nested way too deep